test/conformance/tests/sriov_operator.go

```diff
@@ -913,9 +921,76 @@ var _ = Describe("[sriov] operator", func() {
 				createUnschedulableTestPod(testNode, []string{sriovNetworkName}, resourceName)
 			})
 		})
+
+		Context("NAD update", func() {
+			// 24714
+			It("NAD default gateway is updated when SriovNetwork ipam is changed", func() {
+				resourceName := "sriovnic"
+				sriovNetworkName := "sriovnetwork"
+				testNode := sriovInfos.Nodes[0]
+				sriovDevice, err := sriovInfos.FindOneSriovDevice(testNode)
+				Expect(err).ToNot(HaveOccurred())
+
+				ipam := `{
+					"type": "host-local",
+					"subnet": "10.11.11.0/24",
+					"rangeStart": "10.11.11.171",
+					"rangeEnd": "10.11.11.181",
+					"routes": [{
+					  "dst": "0.0.0.0/0"
+					}],
+					"gateway": "10.11.11.1"
+				  }`
+				err = network.CreateSriovNetwork(clients, sriovDevice, sriovNetworkName, namespaces.Test, operatorNamespace, resourceName, ipam)
+				Expect(err).ToNot(HaveOccurred())
+				Eventually(func() error {
+					netAttDef := &netattdefv1.NetworkAttachmentDefinition{}
+					return clients.Get(context.Background(), runtimeclient.ObjectKey{Name: sriovNetworkName, Namespace: namespaces.Test}, netAttDef)
+				}, 30*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+				Eventually(func() string {
```

Comment:
put this in the eventually above please.

Reply:
Eventuallies merged